bin/jpm

```diff
@@ -69,6 +69,13 @@ program
 program
   .command("xpi")
   .description("Bundle the addon into an .xpi file")
+  .option("--dest-dir <output_dir>", "Where to write the generated xpi",
+          // turn into an absolute path
+          function(dir) {
+            program.xpiPath = path.resolve(dir);
+            return program.xpiPath;
```

Comment:
This looks fragile to me because it relies on a global variable (for example, a refactoring might rename `program` and miss this part). It would be more direct like:

    return path.resolve(dir);

Also, what happens when `--dest-dir` is not given on the command line?

I think instead of making this set `program.xpiPath` that you should modify the signature of `xpi()` so that it looks for the `program.destDir` option.